releasenotes/notes/upgrade-pulse-builder-and-rzx-builder-033ac8ad8ad2a192.yaml

```diff
@@ -0,0 +1,34 @@
+---
+features:
+  - |
+    New pulse directive :class:`~qiskit.pulse.instructions.TimeBlockade` has been added.
+    This instruction is QPY compatible. This directive behaves almost identically to
+    delay instruction, but will be removed before execution.
+    This directive is intended to be used internally within the pulse builder
+    and helps :class:`.ScheduleBlock` represent instructions with
+    absolute time intervals. This allows the pulse builder to convert
+    :class:`Schedule` into :class:`ScheduleBlock`, rather than wrapping with call instruction.
+  - |
+    QPY dump and load now support :class:`~qiskit.pulse.instructions.TimeBlockade` instruction.
```

Comment:
I think we can skip a separate release note for this, it's really part of the introduction of `TimeBlockade`. If you want to explicitly document the new pulse directive also works with qpy I'd just put it in the first feature note that documents the introduction of the `TimeBlockade` class.

Reply:
Fair enough. Done in https://github.com/Qiskit/qiskit-terra/pull/8949/commits/1756e35a9d7ed57101a934ccf0eb6148ba57a31c